ch07/05_dataset-generation/llama3-ollama.ipynb

```diff
@@ -27,7 +27,7 @@
     "tags": []
    },
    "source": [
-    "# Generating An Instruction Dataset via Llama 3 and Ollama"
+    "# Generating Instruction-Response Pair via Llama 3 and Ollama"
```

Comment:
Thanks for all the fixes, but in this particular case, I think the original is more clear.

Reply:
Ah, sorry, this was due to the merge as your commit was pushed after I have submitted my PR (I haven't changed the title).

Reply:
Oh I see! Nvm!